src/routes/docs/quick-starts/solid/+page.markdoc

```diff
@@ -0,0 +1,144 @@
+Create a Vite project.
+
+```sh
+npm create vite@latest my-app -- --template solid && cd my-app
+```
+{% /section %}
+
+{% section #step-3 step=3 title="Install Appwrite" %}
+Install the JavaScript Appwrite SDK.
+
+```sh
+npm install appwrite
+```
+{% /section %}
+
+{% section #step-4 step=4 title="Import Appwrite" %}
+Find your project's ID in the **Settings** page.
+
+{% only_dark %}
+![Project settings screen](/images/docs/quick-starts/dark/project-id.png)
+{% /only_dark %}
+
+{% only_light %}
+![Project settings screen](/images/docs/quick-starts/project-id.png)
+{% /only_light %}
+
+Create a new file `src/lib/appwrite.js` and add the following code to it, replace `<YOUR_PROJECT_ID>` with your project ID.
+
+```js
+import { Client, Account} from 'appwrite';
+
+export const client = new Client();
+
+client
+    .setEndpoint('https://cloud.appwrite.io/v1')
+    .setProject('<YOUR_PROJECT_ID>'); // Replace with your project ID
+
+export const account = new Account(client);
+export { ID } from 'appwrite';
+```
+{% /section %}
+
+{% section #step-5 step=5 title="Create a login page" %}
+Add the following code to `src/App.jsx`.
+
+```js
+import { createSignal } from 'solid-js'
+import { account, ID } from './lib/appwrite';
+
+const App = () => {
+    const [loggedInUser, setLoggedInUser] = createSignal(null);
+    const [email, setEmail] = createSignal('');
+    const [password, setPassword] = createSignal('');
+    const [name, setName] = createSignal('');
+
+    async function login(email, password) {
+        await account.createEmailSession(email, password);
+        setLoggedInUser(await account.get());
+    }
+
+    return (
+        <div>
+            <p>
+                {() => loggedInUser() ? `Logged in as ${loggedInUser().name}` : 'Not logged in'}
+            </p>
+
+            <form>
+                <input type="email" placeholder="Email" value={email()} onChange={e => setEmail(e.target.value)} />
+                <input type="password" placeholder="Password" value={password()} onChange={e => setPassword(e.target.value)} />
+                <input type="text" placeholder="Name" value={name()} onChange={e => setName(e.target.value)} />
+
+                <button type="button" onClick={() => login(email(), password())}>
+                    Login
+                </button>
+
+                <button
+                    type="button"
+                    onClick={async () => {
+                        await account.create(ID.unique(), email(), password(), name());
+                        login(email(), password());
+                    }}
+                >
+                    Register
+                </button>
+
+                <button
+                    type="button"
+                    onClick={async () => {
+                        await account.deleteSession('current');
+                        setLoggedInUser(null);
+                    }}
+                >
+                    Logout
```

Comment:
This button should only be visible if there's a logged in user. Likewise, the form should only exist in case there's no logged in user

Reply:
Hi @TGlide

Thanks so much for these comments. I agree with them and will get working on the updates. 

I didn't want to deviate too far from the React code which is why I made a lot of the decisions in the quick start, just with "Solid" syntax. 

But if you're fine with adding those changes to this quick start and having it deviate a bit from the React version, I'm all for it!